geospaas/base_viewer/forms.py

```diff
@@ -0,0 +1,42 @@
+from django.contrib.gis import forms
+from django.utils import timezone
+from datetime import tzinfo
+from leaflet.forms.widgets import LeafletWidget
+import pytz
+from geospaas.catalog.models import Source
+
+
+class BaseSearchForm(forms.Form):
+
+    polygon = forms.GeometryField(label='choose a location (or leave it empty for all locations)',
+                                  widget=LeafletWidget(attrs={
+                                      'settings_overrides': {
+                                          'DEFAULT_CENTER': (60.0, 5.0),
+                                          'DEFAULT_ZOOM': 1,
+                                          'PLUGINS': {'forms': {'auto-include': True}},
+                                      }
+                                  }))
+    utc = pytz.utc
+    time_coverage_start = forms.DateTimeField(
+        initial=timezone.datetime(2000, 1, 1, tzinfo=utc))
+    time_coverage_end = forms.DateTimeField(initial=timezone.now().utcnow())
```

Comment:
`timezone.now().utcnow()` actually returns the current UTC time as a naive datetime, so it won't do here.
(Plus you call the class method `utcnow()` from the `datetime` class after creating a datetime instance with `now()`, which is inefficient.)
See https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow

You can either:
  - keep `timezone.now()`, which will return an aware datetime if the Django `USE_TZ` setting is set to `True`.
  - if you want to be sure to always get an aware datetime, you can use the `now()` method of the `datetime` module. That would look like this:
```
from datetime import datetime, timezone
[...]
time_coverage_end = forms.DateTimeField(initial=datetime.now(tz=timezone.utc))
[...]
```

Reply:
Thanks. I prefer the first choice. In complementary note to your explanation, as shown in the definition of `now` in Django, using `now` is faster without argument:
![image](https://user-images.githubusercontent.com/62345618/83630774-acc08200-a59c-11ea-8f7c-11a9b0df29bb.png)
So the first choice seems better!

Reply:
OK, but there might be a misunderstanding about the comment in the code: these two ways of creating an aware `datetime` object containing the current UTC time yield the same result:
- `datetime.now(tz=utc)`
- `datetime.utcnow().replace(tzinfo=utc)`

The second one is apparently faster, which is why it is used in `django.utils.timezone.now()`.

But be careful not to confuse `django.utils.timezone.now()` and `datetime.datetime.now()` methods.
When you call `django.utils.timezone.now()` without argument, it will call `datetime.utcnow().replace(tzinfo=utc)` if `USE_TZ` is true (which should be the case for us).
Otherwise, it will call `datetime.datetime.now()` and return a naive `datetime` object.